src/components/Article.jsx

```diff
@@ -1 +1,37 @@
-// export const Article = () => ();
+import React from 'react';
+
+export const Article = ({
+  title,
+  date,
+  text,
+}) => {
+  const dateISO = date.toISOString().slice(0, 10);
+  const dateLocaleString = date.toLocaleDateString('en-US', {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  });
+
+  return (
+    <div className="section">
+      <div className="container">
+        <h1 className="title is-2">React Article</h1>
```

Comment:
This can be moved back to App.js

Reply:
look at where the comment to add an <Article /> started

Reply:
did something change?

Reply:
sorry i forgot to push, my bad